snarkos/store/mod.rs

```diff
@@ -68,6 +68,8 @@ pub enum DataID {
     OutputRecordMap,
     OutputRecordNonceMap,
     OutputExternalRecordMap,
+    // Test
+    Test,
```

Comment:
this variant should be the very last one in order no to perturb the IDs that follow it (we want the tests to be as close to regular runs as possible)